server/remote/gcal/get_default_calendar_view.go

```diff
@@ -21,22 +21,17 @@ const (
 	GoogleEventBusy = "opaque"
 	GoogleEventFree = "transparent"
 
-	ResponseYes   = "accepted"
-	ResponseMaybe = "tentativelyAccepted"
-	ResponseNo    = "declined"
-	ResponseNone  = "notResponded"
-
 	GoogleResponseStatusYes   = "accepted"
 	GoogleResponseStatusMaybe = "tentative"
 	GoogleResponseStatusNo    = "declined"
 	GoogleResponseStatusNone  = "needsAction"
 )
 
 var responseStatusConversion = map[string]string{
-	GoogleResponseStatusYes:   ResponseYes,
-	GoogleResponseStatusMaybe: ResponseMaybe,
-	GoogleResponseStatusNo:    ResponseNo,
-	GoogleResponseStatusNone:  ResponseNone,
+	GoogleResponseStatusYes:   remote.EventResponseStatusAccepted,
+	GoogleResponseStatusMaybe: remote.EventResponseStatusTentative,
+	GoogleResponseStatusNo:    remote.EventResponseStatusDeclined,
+	GoogleResponseStatusNone:  remote.EventResponseStatusNotAnswered,
```

Comment:
Does this break compatibility with mscalendar in anyway?

Reply:
It shouldn't since I didn't modify the `mscalendar` package nor the `remote.Event` struct.

Reply:
Before this change, both provider cases shared the same enum of values in `remote.Event.ResponseStatus`, but now the one returned by the `remote/gcal` package is different than the `remote/mscalendar` package. To be consistent, shouldn't we also change the `mscalendar` case, to use this new set of constants?

Reply:
It's not being _used_ in `msgraph`, but it is implicitly being assigned through the JSON decoding of the msgraph API response. My only concern is that there is core business logic in the plugin that may not match up with what provided by the `remote/msgraph` package, through the msgraph API.

Reply:
We can make it better before we fully merge both plugins, I did a workaround under https://github.com/mattermost/mattermost-plugin-mscalendar/pull/295/commits/4a8d44be35b3d2ac182fae39ea5c66433a61522b